src/sidebar/services/frame-sync.js

```diff
@@ -84,157 +85,156 @@ export class FrameSyncService {
     this._guestRPC = new Bridge();
 
     // Set of tags of annotations that are currently loaded into the frame
```

Comment:
```suggestion
    /**
     * Tags of annotations that are currently loaded into guest frames.
     *
     * @type {Set<string>}
     */
```

- There can be multiple frames
- Specified the type of set elements, since TS can't infer it